lib/activerecord/delay_touching.rb

```diff
@@ -6,9 +6,9 @@ module DelayTouching
     extend ActiveSupport::Concern
 
     # Override ActiveRecord::Base#touch.
-    def touch(name = nil)
+    def touch(*names)
       if self.class.delay_touching? && !try(:no_touching?)
-        DelayTouching.add_record(self, name)
+        DelayTouching.add_record(self, names)
```

Comment:
Hmm technically the code is not validating but just defaulting to nil. The earlier code didn't do any validation of the input either so trying to do it now seems to be conflating the code change. And I will go on a limb here to say typical Ruby coding style lets things blow up instead of trying to validate and massage the arguments. But likely I don't have the big picture of how this fits into the overall stack and the demands on it thereof. 
Overall the change looks good to me (normalize_columns could be renamed tho :))